libs/wingsdk/src/cloud/website.ts

```diff
@@ -52,6 +53,9 @@ export interface WebsiteDomainOptions {
  * @abstract
  */
 export class Website extends Resource implements IWebsite {
+  /** @internal */
+  public [INFLIGHT_SYMBOL]?: IWebsiteClient;
```

Comment:
It would be nice to do something similar to the _toInflightType test, but I don't see an easy way to do it for this. The difference here is that this doesn't exist at runtime, so runtime tests can't validate the existence.

We'd have to write tests that operate on the types. Vitest does have a system for this, but we'd still have to manually enumerate types we want to check.

IMO this would be perfect for an eslint rule instead, but I'll make an issue for that as a followup